app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -48,9 +53,12 @@ abstract class TwoPaneFragment : Fragment() {
             else -> throw IllegalStateException()
         }
 
+    open fun doAfterFolderChanged() {}
```

Comment:
```suggestion
    open fun doAfterFolderChanged() = Unit
```

Reply:
Nope, it's only used in one of both children.

Reply:
Then make it return Unit